src/main/java/cuziFramework/driver/PS2.java

```diff
@@ -0,0 +1,21 @@
+package cuziFramework.driver;
+
+public class PS2 extends PS3 {
+
+    int a;
+
+    public PS2(int a) {
+        super(a);
```

Comment:
Can you add a comment here about how `super` works so that I know that you understand it? Thank you.